app/views/organisations/_contacts.html.erb

```diff
@@ -1,6 +1,6 @@
 <% if contacts.present? %>
 <% contacts.each do |contact| %>
-  <div class="organisation__contact-section <%= "organisation__contact-section--border-top" if local_assigns[:border] %>">
+  <div class="organisation__contact-section <%= "organisation__contact-section--border-top" if local_assigns[:border] %>" <%= "lang=#{contact[:locale]}" if contact[:locale] %>>
```

Comment:
There's a rogue non-English string within this element on line 24. We should either add the lang attribute for the inner element:
```erb
<p lang="<%= :locale %>"><%= t('organisations.contact.email') %></p>
```

or force the `t()` method to retrieve the English version:

```erb
<p><%= t("organisations.contact.email", locale: :en) %></p>
```

(GitHub won't let me comment on line 24, so I'm putting it here.)

Reply:
Thanks for this comment, totally missed that!
 
The problem with `<p lang="<%= :locale %>"><%= t('organisations.contact.email') %></p>` is that it is inaccurate if `t('organisations.contact.email')` does not find a translation for that locale and falls back to the Eng version.

Similarly, `<p><%= t("organisations.contact.email", locale: :en) %></p>` would be no good when the surrounding block is `cy` or some other language.

So in the end I went with `<p lang="en"><%= t("organisations.contact.email", locale: :en) %></p>`

Reply:
Wouldn't you get the same result and less processing with something like this?
<p lang="en"><%= organisations.contact.email %></p>

Reply:
@maxgds I think I know what you mean, but trying `<%= organisations.contact.email %>` throws an error. 
I can't seem to find a precedent where we're pulling data straight from a specific locale file as opposed to using that `t()` function

Reply:
Fair enough - ignore me then.